cob_description/urdf/sensors/realsense_d435.gazebo.xacro

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <!-- use default values of mojin_bringup/d400_rgbd and d435 specs-->
+  <xacro:macro name="realsense_d435_gazebo" params="name
+                                               depth_fps:=15.0 depth_height:=720 depth_width:=1280
+                                               color_fps:=15.0 color_height:=720 color_width:=1280" >
+    <gazebo reference="${name}_link">
+      <sensor type="depth" name="${name}_depth_frame">
+        <always_on>true</always_on>
+        <update_rate>${color_fps}</update_rate>
+        <camera>
+          <horizontal_fov>${91.2*M_PI/180.0}</horizontal_fov>
```

Comment:
horizontal fov should be: 85.2
see here: https://ark.intel.com/products/128255/Intel-RealSense-Depth-Camera-D435